demo/demo_automation.py

```diff
@@ -13,11 +13,7 @@
 
   To run the demo this way, run the following from the main uptane directory
   (which contains, for example, setup.py).
-    python -i demo/demo_automation.py
-
-  That starts the demo in an interactive mode (with a prompt from which
-  you can manipulate commands for the demonstrations). The demo is run in full
-  before any commands can be entered.
+    python demo/demo_automation.py
```

Comment:
Don't forget to update the pull request summary to match the change.


What's current in the summary:
```
Running python -i demo/demo_automation.py executes all the
commands in the console demo, along with notes about expected
output. I'll make this into a more useful test script next.
```

Reply:
Surely will, thanks :)